htsget_server/beacon_operations.py

```diff
@@ -0,0 +1,600 @@
+from flask import Flask
+import variants
+import drs_operations
+import htsget_operations
+import variants
+import database
+import json
+import re
+import connexion
+
+
+app = Flask(__name__)
+
+API_VERSION = '1.0.0'
+BEACON_ID = 'org.candig.htsget.beacon'
+SCHEMA = [
+    {
+        "entityType": "genomicVariant",
+        "schema": 'ga4gh-beacon-variant-v2.0.0'
+    }
+]
+
+# Endpoints
+def get_beacon_service_info():
+    return {
+        "id": BEACON_ID,
+        "name": "CanDIG Beacon v2 genomic variants service",
+        "type": {
+            "group": "org.ga4gh",
+            "artifact": "beacon",
+            "version": "v2.0.0"
+        },
+        "description": "A Beacon v2 server for CanDIG genomic data",
+        "organization": {
+            "name": "CanDIG",
+            "url": "https://www.distributedgenomics.ca"
+        },
+        "version": API_VERSION
+    }
+
+
+# req = {
+#     include_result_set_responses: string,
+#     pagination: {
+#         currentPage: string,
+#         limit: int,
+#         nextPage: string,
+#         previousPage: string,
+#         skip: int
+#     }
+#     requestParameters: {
+#       ...
+#     }
+#     requestedGranularity: string,
+#     testMode: boolean
+# }
```

Comment:
It is compiled from the documentation in beacon_openapi.yaml. I just copied it over to the python file so it would be easier for me to see as I was coding. Should I take it out?